scripts/run_e2e_tests.py

```diff
@@ -607,9 +648,11 @@ def main(args=None):
             break
         flaky = flake_checker.is_test_output_flaky(
             output, parsed_args.suite)
-        # Don't rerun if the test was non-flaky and we are not
-        # rerunning non-flaky tests.
-        if not flaky and not RERUN_NON_FLAKY:
+        # Check whether we should rerun based on this suite's policy.
+        policy = RERUN_POLICIES.get(parsed_args.suite, 'always')
```

Comment:
Any specific reason for adding default value here?

Reply:
removed